packages/gestalt/src/Module/applyModuleDensity.js

```diff
@@ -0,0 +1,26 @@
+// @flow strict
+import Box from '../Box.js';
+
+type Gap = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+
+const applyModuleDensityStyle = (
+  size: 'sm' | 'md' | 'lg',
+): {
+  gap: Gap,
```

Comment:
this is not really 0-12 it's more like 2 | 4 | 6

That or `$ElementType<React$ElementConfig<typeof Flex>, 'gap'>`

Reply:
This causes flow errors.

The Gap prop on Flex supports two kinds of values, a row/col object, or numerical Gap. In this case, I only want the numerical type

 <Flex direction="column" gap={{ column: gap, row: 0 }}>